cli/upload.py

```diff
@@ -159,43 +165,42 @@ def _start_procs(upload_info: api.UploadInfo, xlsx: str) -> list:
 
 def _wait_for_upload(procs: list) -> bool:
     """
-    Waits for all subprocesses and prints their stderr streams.
+    Waits for all subprocesses and click.echos their stderr streams.
     Returns bool - if an error has occurred during any if uploads
     """
 
     finished = set()
     errored = False
+    progress_bar = "."
     while len(finished) != len(procs) and not errored:
-        for i, p in enumerate(procs):
 
+        click.secho(f"Uploading files ({len(finished)} / {len(procs)} done)", bold=True)
+        click.echo(progress_bar)
+        for i, p in enumerate(procs):
+            click.secho(f"(proc {i + 1}) ", nl=False, fg="bright_blue")
             if i in finished:
+                click.secho("uploaded ", nl=False, fg="green", bold=True)
                 continue
+            else:
+                click.echo("uploading ", nl=False)
 
             if p.poll() != None:
                 finished.add(i)
 
                 if p.returncode != 0:
                     errored = True
 
-            errline = p.stderr.readline()
+            click.echo(p.args[-2])
 
-            # skipping "large file" warnings
-            if errline.strip() in _IGNORED_WARN_LINES:
-                continue
+        time.sleep(1)
```

Comment:
why is that needed here?

Reply:
Since we print every time we poll, this reduces the amount of output the CLI generates. Got any arguments against `sleep`ing? I don't feel strongly.

Reply:
I don't see how sleeping reduces output? 
1. Pool will sleep for until there's an output to pass
2. We still need to print everything we get

so i think this just hangs printing for a second, not changing other behavior - thus the question why would we need it